mlrun/datastore/targets.py

```diff
@@ -1834,6 +1788,7 @@ class CustomTarget(BaseStoreTarget):
     is_online = False
     support_spark = False
     support_storey = True
+    support_pandas = True
```

Comment:
This doesn't look right.
```suggestion
    support_pandas = False
```

Reply:
was True before, remove please:)